src/components/feedback-modal/dialogs/thank-you.tsx

```diff
@@ -0,0 +1,49 @@
+import { TypographyScale, Button } from '@mdb/flora';
+
+import { modalWrapperStyles } from '../styles';
+
+const ThankYou: React.FunctionComponent<{ onContinue: () => void }> = ({
+    onContinue,
+}) => {
+    const title = 'We appreciate your feedback.';
+    const subtitle = (
+        <>
+            We&apos;d love to chat with you and answer your questions in our
+            online{' '}
+            <a
```

Comment:
Just wanted to double-check we don't want to use the Link component for this?

Reply:
Ah yeah so I actually ran into the same issue as Harika with the Links not being inline, so I just did this, but I'll try to fix it, thanks for catching.

Reply:
Actually just gonna merge and fix this in my next PR.